etc/adminhtml/system.xml

```diff
@@ -141,6 +141,10 @@
                     <label>Use Store Credit on Shopping Cart</label>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                 </field>
+                <field id="enable_payment_only_checkout" translate="label" type="select" sortOrder="250" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
shouldn't this be inferred via presence of paymentonly publishable key?

Reply:
@daisy1754 Yes but the solution was front-end based and had an error as well so we had issues with checkout that is not the default Magento one-page checkout but 3rd party (Magewrox one-step checkout on Coastal Business). This way it is removed from layout completely.